.gitattributes

```diff
@@ -2,5 +2,5 @@
 * text=auto
 # Explicitly declare text files we want to always be normalized and converted
 # to native line endings on checkout.
-*.md text
+*.md text diff=markdown
```

Comment:
Did you intend to commit this change to the repository? If so, could you tell me why you'd like to make this change.

Reply:
Yes, I very much ment to commit this change :) I explained why in the commit message of my 4th commit, https://github.com/github/docs/pull/16968/commits/551bf0a62fae1625c65f1eb76359fe292cb95006, which I copy verbatim here: 

~~~
.gitattributes: mark '*.md' as Markdown files

Denote Markdown files as such with the 'diff' Git attribute [1]. This
allows various Git commands (diff [2, 3], grep [4, 5], log [6], blame
[7]) to show in which part of the "code" the changes are located in the
hunk header.

The history of the "Using Git" section modified in the previous commit
specific entry can now be interrogated using

    git log -L:"Using Git":content/get-started/quickstart/git-and-github-learning-resources.md

[1] https://git-scm.com/docs/gitattributes#_defining_a_custom_hunk_header
[2] https://git-scm.com/docs/git-diff#_generating_patch_text_with_p
[3] https://git-scm.com/docs/git-diff#Documentation/git-diff.txt--W
[4] https://git-scm.com/docs/git-grep#Documentation/git-grep.txt--p
[5] https://git-scm.com/docs/git-grep#Documentation/git-grep.txt--W
[6] https://git-scm.com/docs/git-log#Documentation/git-log.txt--Lltfuncnamegtltfilegt
[7] https://git-scm.com/docs/git-blame#Documentation/git-blame.txt--Lltfuncnamegt
~~~

Reply:
Terrible message I did not want to leave terrible message to run rampid  yours Truely Brian Donnellan.com

Reply:
Hi @phil-blain - thanks for copying the commit message into this conversation. 

We rely on the information in the pull request itself, rather than doing commit-by-commit reviews, so I'd missed that extra context. This change will need confirmation from a site engineer.

Reply:
We took a look and we're ok with this change 👍 !